README.md

```diff
@@ -12,6 +12,22 @@
 
 OpenSearch Continous Integration is an open source CI system for OpenSearch and its plugins.
 
+## Getting Started
+
+- Requires [NPM](https://docs.npmjs.com/cli/v7/configuring-npm/install) to be installed
+- [Install CDK](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) using `npm install -g aws-cdk`
```

Comment:
Curious why this isn't install via `npm install`?

Reply:
It is installed, but this was a personal preference, I'll remove this step and update with `npm run cdk ...` so admin/sudo isn't needed for onboarding

Reply:
Resolved in #13